src/views/Compose.vue

```diff
@@ -172,6 +172,19 @@
                   </div>
                 </ocs-custom-field>
               </template>
+              <template #target-fields-footer="slotProps">
+                  <span v-show="showFractionalRate(slotProps.data.target)">
```

Comment:
This function is used since it has a side-effect of also clearing out the `fractional_ephemeris_rate` if we switch off a non-major planet orbital elements type, since we don't want to send the parameter in that case.

Reply:
Would this be worth adding as a comment?

Reply:
I agree with adding a comment like Tim said - you could also remove the fractional ephemeris rate from the target.extra_params in a callback when the user switches the target type? 

https://vuejs.org/v2/guide/events.html#Method-Event-Handlers

Reply:
I did it this way because I don't think you can do a simple callback. The problem is that the thing you would need to do a callback on is in the ocs-component-lib component and not accessible in this frontend directly, so you would need to add a callback in the ocs-component-lib that then calls some placeholder method, and then extend the component in the frontend here to override that method to do our custom stuff. Seemed to extreme and not applicable to the general ocs so I didn't want to add it.